bin/node/runtime/src/lib.rs

```diff
@@ -884,6 +876,15 @@ impl pallet_bounties::Config for Runtime {
 	type DataDepositPerByte = DataDepositPerByte;
 	type MaximumReasonLength = MaximumReasonLength;
 	type WeightInfo = pallet_bounties::weights::SubstrateWeight<Runtime>;
+	type ChildBountyManager = ChildBounties;
+}
+
+impl pallet_child_bounties::Config for Runtime {
```

Comment:
Why do we need this pallet?

Reply:
We use `pallet-bounties` and `pallet_child_bounties` [tightly coupled](https://paritytech.github.io/substrate/master/pallet_child_bounties/index.html) with it.